### DISCUSSION TITLE
Hottop 2K+ connection issues

### DISCUSSION THREAD

**Discussion by Jackroast** [artisan-roaster-scope/artisan#1679] [Q&A]
Hi everyone. I'm trying to connect a recently purchased second-hand Hottop 2K+ to artisan. Here's where I'm at: I've downloaded version 3.0.2, I selected the Hottop KN-8828B-2K+ machine, I selected the "COM3" port, I turned on my machine, I have pressed every combination of buttons in artisan to start my machine and I've had no luck. Under "Device Manager" settings, my communications port has a warning triangle with a Code 34 error saying windows can't determine the settings for this device. My laptop only has usb-c ports and I'm not sure which one corresponds with "COM3" but I'm using a USB transfer cable + usb->usb-c adapter to try and connect my roaster to my laptop. Has anyone experienced this issue before? There's very little information on this. I tried downloading an older version of artisan with a separate VCP FTDI serial driver and I'm still having no luck. Any suggestions to fix are much appreciated!!
![image](https://github.com/user-attachments/assets/8f64f92d-31b0-4fc5-a7ff-b4edc65d44fb)

**Comment by Jackroast** [comment]
![image](https://github.com/user-attachments/assets/6f225c78-13a1-4ce0-94ea-a6952d743ba1)
![image](https://github.com/user-attachments/assets/0d16e165-d077-4683-9e1d-4aef02aec6c9)

**Comment by MAKOMO** [comment]
What was the issue and how did you solve it?

**Reply by Jackroast** [reply]
The seller didn't include the original usb connector. I contacted hottop and bought an OEM one.
